README.md

```diff
@@ -8,22 +8,24 @@ If you have any issues using the API or would like some help, please open [an is
 
 ## API Overview
 
-The API tends to be RESTful and returns results in JSON by default.
+The API is generally RESTful and returns results in JSON by default.
 This means for example that it will return a 404 HTTP status code if a record can't be found.
 
-In general, any page you get through the website has a corresponding representation in the API obtained by prefixing its URL with `/api/`.
+In general, most pages you get through the website have a corresponding representation in the API obtained by prefixing the path component of the URL with `/api/`.
 For example, the data displayed at
 ```
-https://inspirehep.net/literature?sort=mostrecent&size=25&page=1&q=title%20api
+https://inspirehep.net/literature?sort=mostrecent&size=25&page=1&q=title api
 ```
 is available through the API at
 ```
-https://inspirehep.net/api/literature?sort=mostrecent&size=25&page=1&q=title%20api
+https://inspirehep.net/api/literature?sort=mostrecent&size=25&page=1&q=title api
 ```
 
+Currently only read-only operations are allowed and they all use the `GET` HTTP method.
+
 ## Rate limiting
 
-In order to avoid overwhelming the server, we enforce rate limits per IP address: every IP address is allowed 50 requests, then at most 2 requests per second. If you exceed those limits, you will receive a response with HTTP status code 429.
+In order to avoid overwhelming the server, we enforce rate limits per IP address: every IP address is allowed 50 requests, then at most 2 requests per second. If you exceed those limits, you will receive a response with HTTP status code 429 and a `x-retry-in` header telling you how long to wait before retrying.
```

Comment:
@tsgit commented:
>the API should send the relevant info in the corresponding headers
and I would simply demand that any client application follow the generally accepted rules
with a link to rate limiting docs https://developer.github.com/v3/#rate-limiting
or any number of similar links

The issue here is that traefik doesn't provide the rate limiting headers before you reach the limit. It will only set `Retry-after: 0` and `x-retry-in: 500ms` in the 429 response (I believe this is due to the leaky bucket model it uses, that basically allows you to make an extra request every .5s with our current settings after reaching the burst limit of 50 requests).

@tsgit also commented:
> rate limit solely by IP address risks blocking entire institutions behind a
proxy due to one bad player. maybe also include user-agent or other
signatures of the client
>
>In fact I would add a section on setting some informative User-Agent and
contact info in the Headers so that Inspire can contact the user in case of
problems (most problems are inadvertent, the bad players will always fake
this)

These are good points, however traefik can only rate limit by a single function (see https://docs.traefik.io/v1.7/configuration/commons/#rate-limiting) and is a black box, so we can't take the user agent into account nor put clients in some whitelist if they match some criteria. Using tokens would have been more flexible in this respect, but it was deemed too complex to implement for the first version of the API.